preprocessors/semanticSeg/segment.py

```diff
@@ -72,15 +73,18 @@ def findContour(pred_color, width, height):
     ret, thresh = cv2.threshold(gray_image, 10, 255, cv2.THRESH_BINARY)
     contours, hierarchy = cv2.findContours(
         thresh, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_NONE)
+    logging.info("drawing all contours")
     cv2.drawContours(image, contours, -1, (0, 255, 0), 2)
     # removes the remaining part of image and keeps the contours of segments
+    logging.info("deleting remainder of the image except the contours")
     image = image - dummy
     centres = []
     area = []
     totArea = 0
     send_contour = []
     flag = False
     # calculate the centre and area of individual contours
+    logging.info("computing individual contour metrics")
```

Comment:
suggest including len(contours) here so we can see if there are an outrageous number of iterations here